hyperpoint.cpp

```diff
@@ -21,8 +21,8 @@ namespace hr {
 static constexpr ld A_PI = M_PI;
 static constexpr ld TAU = 2 * A_PI;
 static constexpr ld degree = A_PI / 180;
-static constexpr ld golden_phi = (sqrt(5)+1)/2;
-static constexpr ld log_golden_phi = log(golden_phi);
+static constexpr ld golden_phi = noexcept((sqrt(5)+1)/2);
+static constexpr ld log_golden_phi = noexcept(log(golden_phi));
```

Comment:
Probably obvious to @zenorogue, but just in case: obviously don't merge this change, unless you're OK with `golden_phi` becoming `0.0`. 😛

Reply:
oh wait